pysqldb3/pysqldb3.py

```diff
@@ -807,6 +807,10 @@ def dataframe_to_table_schema(self, df, table, schema=None, overwrite=False, tem
         for col in df.dtypes.items():
             col_name, col_type = col[0], type_decoder(col[1], varchar_length=allowed_length)
 
+            # autodetect date and force to text (common error)
+            if 'date' in col_name.lower() and col_type in ('int', 'bigint', 'float'):
```

Comment:
Not sure about this but Do all the columns with date type data in ris have 'date' in their name?

Reply:
no, it could miss some, but it was missing some date fields and I figured this was a relatively safe assumption, since worse case it stores as text which will prevent errors but not introduce any new ones. If it tries as int and finds a date it will fail, but if it assumes it to be text it will always work